controller/src/controller/services/builder/lab.py

```diff
@@ -670,10 +678,24 @@ def _build_pod_env_volume(self, username: str) -> MountedVolume:
             ),
         )
 
-    def _build_pod_tmp_volume(self) -> MountedVolume:
-        """Build the volume that provides a writable tmpfs :file:`/tmp`."""
+    def _build_pod_tmp_volume(self, mem_size: int) -> MountedVolume:
+        """Build the volume that provides a writable tmpfs :file:`/tmp`.
+
+        Note that it is the `Memory` medium setting that forces this to
+        be tmpfs rather than ephemeral storage, and therefore usage will
+        come from the pod memory allocation rather than disk space.
+        """
+        medium: str | None = None
```

Comment:
This should work fine without the type annotation. mypy recognizes this construction.
```suggestion
        medium = None
```